test/spec/services/applicationGeneratorSpec.js

```diff
@@ -3,13 +3,23 @@
 describe("ApplicationGenerator", function(){
   var ApplicationGenerator;
   var inputTemplate;
+  var mathFloor = Math.floor;
 
   beforeEach(function(){
 
     inject(function(_ApplicationGenerator_){
       ApplicationGenerator = _ApplicationGenerator_;
-      ApplicationGenerator._generateSecret = function(){
```

Comment:
Is there a reason we can't simply change this to `ApplicationGenerator.generateSecret` and leave the mock as before?